src/main/java/com/fc/invoicing/services/InvoiceService.java

```diff
@@ -0,0 +1,73 @@
+package com.fc.invoicing.services;
+
+import com.fc.invoicing.db.Database;
+import com.fc.invoicing.db.JpaInvoiceRepository;
+import com.fc.invoicing.dto.InvoiceListDto;
+import com.fc.invoicing.dto.mappers.InvoiceListMapper;
+import com.fc.invoicing.exeptions.handlers.ItemNotFoundExemption;
+import com.fc.invoicing.model.Invoice;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class InvoiceService implements Database<Invoice> {
```

Comment:
implements Database is not needed here anymore

Reply:
This is just service, should not have nothing in common with db

Reply:
changed